### PR TITLE
Add Tap Release to the list of apps

### PR DESCRIPTION
**Pull request by toolmantim**
This adds https://github.com/toolmantim/tap-release to the list of apps 🎉

### CODE REVIEW

**Comment by toolmantim** [comment]
Okay, looks like the CI failure now isn’t anything to do with this PR.

**Comment by toolmantim** [comment]
Thanks for the review @hiimvex.

I was avoiding duplicating it a third time (it’s also in the GitHub app description) but I’ll just copy+paste it for now anyway.

I’ll just have to remember to PR this repo every time I update the readme.

**Comment by hiimbex** [comment]
👍 @toolmantim we've thought about ways to automate this process and pull from your app's readme but everyone does it differently.

> but I’ll just copy+paste it for now anyway

I think you might've forgotten to do this :)

**Comment by toolmantim** [comment]
Thanks for the reminder @hiimbex. I've just pushed the full readme.

I'm guessing I have to remove/update all of the relative URLs in the markdown as well yeah? Such as `[CONTRIBUTING.md](CONTRIBUTING.md)` and `[URL variables](#url-variables)`?

**Comment by toolmantim** [comment]
Is the intention that people never have to go to the GitHub repo to use the app, or is linking to the GitHub repo for all the configuration options okay?

If linking to GitHub is okay, I can probably just leave off all the configuration options and final sections, and have a paragraph that reads:

> For a list of all the configuration options and template variables, instructions for running your own copy of Tap Release, and how to contribute, see the [Tap Release README on GitHub](https://github.com/toolmantim/tap-release#readme).

**Comment by hiimbex** [comment]
> Is the intention that people never have to go to the GitHub repo to use the app, or is linking to the GitHub repo for all the configuration options okay?

I'm okay with not having all the configuration listed on the site, but I'd prefer to have some, or a smaller example configuration listed, so folks can see what using the app would look like from the get go and get a better feel for how they app will work.

Like maybe some shortened form of this section: https://github.com/toolmantim/tap-release#configuration-options would be helpful? I just want folks who wouldn't be familiar at all with Homebrew taps (read: me) to understand how this would work, at least at a very high level. 

> 'm guessing I have to remove/update all of the relative URLs in the markdown as well yeah? Such as [CONTRIBUTING.md](CONTRIBUTING.md) and [URL variables](#url-variables)?

Yes, but let's agree on a simpler version of the readme before you going changing all of those since that's a pain :)

**Comment by toolmantim** [comment]
Thanks for the feedback!

I’m working on another GitHub app at the moment. When I get a chance I’ll boot up the site locally and test this PR properly, and get the others ready for submission too.

**Comment by toolmantim** [comment]
Finished up https://github.com/toolmantim/release-drafter and was able to circle back to this and get it running locally, so I could check how it looks on the site.

I've just cleaned up the commit, and pushed a new version. Could you check it out again @hiimbex?

File: _apps/tap-release.md [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+host: https://tap-release.now.sh
+---
+
+## Usage
+
+Firstly, you’ll need to install the [Tap Release GitHub App](https://github.com/apps/tap-release). This listens out for any releases, or any changes to the configuration.
+
+Then, add a `.github/tap-release.yml` configuration file to the GitHub repository where you publish new releases to.
+
+For example, given the following `.github/tap-release.yml` file in a `my-org/app/app.rb` repository:
```
Comment (hiimbex) [inline _apps/tap-release.md]: > `my-org/app/app.rb` repository

I see further down the importance of the app.rb file, but the usual syntax for referring to a repo is just owner/repo, so maybe that would be more clear here?

Comment (toolmantim) [reply _apps/tap-release.md]: Oh, that was a straight up typo! Thanks for the catch. All fixed!

**Comment by hiimbex** [comment]
Wooo! Thanks @toolmantim ! Sorry this took so long to get out

**Comment by toolmantim** [comment]
Thanks for the help getting it over the line ☺️🙏🏼

**Pull request merged**